docs/pages/heading.js

```diff
@@ -99,64 +335,78 @@ export default function DocsPage({ generatedDocGen }: {| generatedDocGen: DocGen
-      次の単語グレートブリテンおよび北アイルランド連合王国で本当に大きな言葉
-    </Heading>
-  </Box>
+    <Box color="secondary" padding={2} rounding={2}>
+      <Heading size="400" overflow="breakWord">
+        This is a long and Supercalifragilisticexpialidocious sentence.
+        次の単語グレートブリテンおよび北アイルランド連合王国で本当に大きな言葉
+      </Heading>
+    </Box>
+  </Flex>
 
-  <Box paddingY={2}>
+  <Flex direction="column" gap={2}>
     <Text>normal:</Text>
-    <Heading size="400" overflow="normal">
-      This is a long and Supercalifragilisticexpialidocious sentence.
-      次の単語グレートブリテンおよび北アイルランド連合王国で本当に大きな言葉
-    </Heading>
-  </Box>
+    <Box color="secondary" padding={2} rounding={2}>
+      <Heading size="400" overflow="normal">
+        This is a long and Supercalifragilisticexpialidocious sentence.
+        次の単語グレートブリテンおよび北アイルランド連合王国で本当に大きな言葉
+      </Heading>
+    </Box>
+  </Flex>
 
-  <Box paddingY={2}>
-    <Text>lineClamp:</Text>
-    <Heading size="400" lineClamp={2}>
-      This is a long and Supercalifragilisticexpialidocious sentence.
-      次の単語グレートブリテンおよび北アイルランド連合王国で本当に大きな言葉
-    </Heading>
-  </Box>
-</Box>
+  <Flex direction="column" gap={2}>
+  <Text>lineClamp:</Text>
+    <Box color="secondary" padding={2} rounding={2}>
+      <Heading size="400" lineClamp={2}>
+        This is a long and Supercalifragilisticexpialidocious sentence.
+        次の単語グレートブリテンおよび北アイルランド連合王国で本当に大きな言葉
+      </Heading>
+    </Box>
+  </Flex>
+</Flex>
 `}
-      />
-      <Example
-        id="align"
-        name="Example: Alignment"
-        defaultCode={`
-<Box>
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection title="Alignment">
+          <MainSection.Card
+            cardSize="lg"
+            defaultCode={`
+<Flex direction="column" gap={4}>
   <Heading align="start" size="400">Start-aligned heading (default)</Heading>
+  <Divider />
   <Heading align="end" size="400">End-aligned heading</Heading>
+  <Divider />
   <Heading align="center" size="400">Center-aligned heading</Heading>
+  <Divider />
   <Heading align="justify" size="400">Justify-aligned heading</Heading>
+  <Divider />
   <Heading align="forceLeft" size="400">Forced-left-aligned heading</Heading>
+  <Divider />
   <Heading align="forceRight" size="400">Forced-right-aligned heading</Heading>
-</Box>
+</Flex>
 `}
-      />
-      <Example
-        id="levels"
-        description="
-    For accessibility purposes, we allow you to override the heading level.
-
-    We should have one level 1 per page &amp; levels should be appropriately nested. E.g. level 1 followed by level 2 &amp; level 2 followed by level 2 or level 3. We also allow headings without an accessibility level.
-  "
-        name="Example: Levels"
-        defaultCode={`
-<Box>
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection
+          title="Accessibility level"
+          description="For accessibility purposes, we allow you to override the heading level.
+
```

Comment:
we allow you to override the heading level. Any guidance on why Heading with no level vs just Text weight="bold"?

I think the none value is legacy, maybe we could not mentioned it. Or maybe mention is legacy and Text is preferred